galaxy.yml

```diff
@@ -12,7 +12,6 @@ dependencies:
   community.crypto: ">=2.2.3"
   community.general: ">=4.5.0"
   kubernetes.core: ">=2.3.2"
```

Comment:
Can you make sure if those collections are being used?

You can CTRL+F `ansible.posix` and see if there is any matches and do the same for the rest, so we can remove the extra dependencies :)